recipes/auditd.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+#
+# Cookbook Name: os-hardening
+# Recipe: auditd.rb
+#
+# Copyright 2017, Artem Sidorenko
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package 'auditd'
```

Comment:
Based on the CI tests I noticed the different names here. I'll add a fix to this PR